10up/distributor
# Allow user clicks on confirm dialog buttons

### Description of the Change
This change prevents locking out the buttons in Gutenberg confirmation dialogs, by reverting the `opacity`, `pointer-events` and `cursor` values set for distributed posts.
Closes #990 

### How to test the Change
Created a post on source/original site, and distribute to the destination site. When publishing to the destination site, unchecking the "Save as Draft" option. Attempt to switch back to draft on the destination site - confirmation modal buttons are disabled. After applying this PR, the modal buttons should be enabled.

### Changelog Entry
Fixed - Allow clicking on the `Save to draft` confirmation modal buttons

### Credits
Props @av3nger 

### Checklist:
